src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -86,55 +85,44 @@ public class SendReceiveTest : ClientWebSocketTestBase
         {
             using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
             {
-                var cts = new CancellationTokenSource(TimeOutMilliseconds);
+                var tasks = new Task[2];
```

Comment:
Why did you change this from 10 to 2?  Part of the goal of this test is to verify behavior when there are a bunch of outstanding send operations.  Having just one outstanding significantly diminishes the value of what's being tested.